Cargo.toml

```diff
@@ -23,3 +23,12 @@ anyhow = "1.0"
 rand = "0.8"
 arbitrary = "1.3"
 console = "0.15"
+
+[workspace.lints.clippy]
+all = { level = "deny", priority = 0 }
```

Comment:
This is usually not a good idea. -Dwarnings in CI does everything this does (you can't merge without fixing the lints), but this makes developing more annoying since you can't build *locally* without fixing the lints.